handlers/map-tactile-svg/map-svg.py

```diff
@@ -355,5 +361,75 @@ def getDescriptions(street):
         return description[:-2]
 
 
+def getNodeDescription(POI, checkPOIs):
+    label = ""
+    drawPOI = False
+    if "intersection" in POI:
+        drawPOI = True
+        label += "Intersection of "
+        if len(checkPOIs[POI["id"]]) == 1:
+            label += checkPOIs[POI["id"]][0][0]+" and minor street"
```

Comment:
To check, do these intersections include ways such as footpaths? Just trying to anticipate if someone may end up getting confused by "street" here.

Reply:
The intersections include the filtered-out _street_type_s like "footway", "crossing", "steps" (and "elevator") but call them "minor street" as long as there is at least one street of _street_type_ that is not filtered out at that intersection. (If all the streets at an intersection have _street_type_s that are filtered out it is not rendered on the SVG.)
I have logged an issue #706 for changing this tag to just "Intersection" (mostly for brevity rather than to specifically address this issue though).

Reply:
I don't particularly like the use of `minor street` for features that are not streets, but if you're changing this behavior shortly I can live with it.